snapd.code-workspace

```diff
@@ -0,0 +1,50 @@
+{
+	"folders": [
+		{
+			"path": "."
+		}
+	],
+	"settings": {
+		"go.testEnvVars": {"LANG": "C.UTF-8", "SNAPD_DEBUG": "1"},
```

Comment:
as we discussed the test are not currently designed to fully work with SNAPD_DEBUG=1, not sure setting it here make sense